README.md

```diff
@@ -1,2 +1,23 @@
-# Apple_Clone
-Microverse Solo Project
+# micro-apple
+Microverse Project
+
+Backgrounds and Gradients 
+
+This project is to learn how to apply backgrounds and gradients with HTML and CSS.
+
+The task was to copy apple's old website apple website
+
```

Comment:
The presentation of the readme file is really important since it's the first thing people will see when checking your projects, and it's a good skill to build along the Microverse journey, the more you work on that, the better you will be at it and with more ease you will be able to do great readme files that make you look like a serious professional. So, please, haven into account [this template](https://github.com/othneildrew/Best-README-Template) and try to add some changes in this project or moving on.